### PR TITLE
Update AboutUs

### LINKED ISSUE #42: [Bryan] Update AboutUs Page
https://nus-cs2103-ay2223s1.github.io/website/schedule/week7/project.html#2-update-project-website-aboutus-readme

### PR DESCRIPTION
**Pull request by bryanljx**
Depends on #81, please review and merge #81 first

Closes #42

### CODE REVIEW

**Review by RichDom2185 — CHANGES_REQUESTED** [review]
Please rename `bryanljx.jpg` to `bryanljx.png` to follow module requirements.

**Comment by bryanljx** [comment]
> Please rename `bryanljx.jpg` to `bryanljx.png` to follow module requirements.

Done

File: docs/AboutUs.md [hunk 1]
```diff
@@ -9,6 +9,15 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
+### Bryan Lim Jing Xiang
+
+<img src="images/bryanljx.jpg" width="200px">
```
Comment (RichDom2185) [inline docs/AboutUs.md:14]:
```suggestion
<img src="images/bryanljx.png" width="200px">
```

**Review by RichDom2185 — CHANGES_REQUESTED** [review]

**Review by RichDom2185 — APPROVED** [review]
LGTM!

**Pull request merged**